### DISCUSSION TITLE
domain is used by another account

### DISCUSSION THREAD

**Discussion by metrue** [vercel/community#5097] [Bug Report]
### Page to Investigate

https://vercel.com/metrue/collections-pub/settings/domains

### Steps to Reproduce

I am trying to add domain '*.iiiii.li' to my project 'collection.pub', but failed with '"*.iiiii.li" does not exist.'?

![Screenshot 2023-12-07 at 14 33 54](https://github.com/vercel/community/assets/1001246/93e0adbc-2f04-4226-add7-9dfa7929b0a6)

**Comment by ghost** [comment]
I don't believe you can link a subdomain directly. Can you try entering iiii.li please?

**Reply by metrue** [reply]
I did add the entire `iiiii.li`.
<img width="940" alt="Screenshot 2023-12-09 at 16 12 54" src="https://github.com/vercel/community/assets/1001246/b7e852ee-3fb3-4fe9-8e56-1d5657fa1f14">


Now, I am trying to add the vercel dns name server for it and see.
